src/preprocess/srtm.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from shutil import rmtree
+import xarray as xr
+import os
+
+from typing import Optional
+
+from .base import BasePreProcessor
+
+
+class SRTMPreprocessor(BasePreProcessor):
+    dataset = 'srtm'
+    static = True
+
+    def regrid(self, ds: xr.Dataset, regrid: Path, method: str = 'remapbil') -> xr.Dataset:
```

Comment:
are we happy with bilinear? i guess it shouldn' matter too much because we're going HIGH -> LOWER resolution. I only worry about it when we're making variance up using the bilinear methods from low->high resolution to be fair